### ISSUE TITLE
[BUG] Add new nestId syntax to the 1.4 dev notes

### ISSUE THREAD

**Issue by Aliharu** [Larkinabout/fvtt-token-action-hud-core#128]
Right now the 1.4 dev notes do not mention that the nestId syntax was changed from parent_id_child_id to parent-id_child-id.  This has already caused blocks on me doing Exalted Third and Essence20 based off this issue #127 .  I was only able to figure out what was going on through the Essence20 issue.

**Comment by Larkinabout** [comment]
It hasn't changed in 1.4 specifically, the nestId has always been a chain of the nested group ids delimited by an underscore. I'll add  clarification in the 'Developing a System Module'.

**Comment by Aliharu** [comment]
Yes but in 1.3 parent_id_child_id worked as a nestId when in 1.4 its not so its technically a breaking change.  Having it in the dev notes is fine though.
